### PR TITLE
LPS-77063 Change span to a tag

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:retest

**Comment by alinedoleron** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy [hunk 1]
```diff
@@ -115,15 +115,25 @@
...
 					<div class="drop-chosen {$open ? '' : 'hide'}">
-						<div class="search-chosen">
-							<div class="select-search-container">
-								{if $selectSearchIcon}
-									<a class="" href="javascript:;">{$selectSearchIcon}</a>
-								{/if}
-							</div>
-							<input autocomplete="off" class="drop-chosen-search" placeholder="{$strings.search}" type="text">
+						<div class="dropdown-menu" aria-labelledby="theDropdownToggleId">
+							<forms>
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: @alinedoleron Is this `<forms>` tag correct?

Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: OMG! Thank you, @brunobasto! It should be <form>.

**Comment by alinedoleron** [comment]
sent to https://github.com/rafaprax/liferay-portal/pull/302

**Pull request closed without merging**